packages/ppx/test/bucklescript/src/selectors_test.re

```diff
@@ -19,7 +19,7 @@ let testData = [
     CssJs.style(.
       [|
         CssJs.selector(.
-          {js|&:nth-child(even)|js},
+          {js|& :nth-child(even)|js},
```

Comment:
I believe this isn't valid right?

Reply:
i tested in the demo using:
```
[%styled.div {|
  &:nth-child(even){
    display: none;
  }
|}]
```

that output this, and it works